client/interviews-service.js

```diff
@@ -0,0 +1,17 @@
+const instance = require('./axios-instance')
+
+class InterviewsService {
```

Comment:
Class declaration is not needed here. You even not using context. This methods are static